packages/ui/src/proposals/modals/AddNewProposal/AddNewProposalModal.tsx

```diff
@@ -70,6 +71,7 @@ export const AddNewProposalModal = () => {
   const maximumReferralCut = api?.consts.members.referralCutMaximumPercent
   const minCashoutAllowed = api?.consts.content.minimumCashoutAllowedLimit
   const maxCashoutAllowed = api?.consts.content.maximumCashoutAllowedLimit
+  const palletFrozenStatus = useObservable(() => api?.query.projectToken.palletFrozen(), [api?.isConnected])
```

Comment:
Because this subscription doesn't need to stay opened please use `useFirstObservableValue` instead